Notebooks/Russian Twitter Corpus/Untitled.ipynb

```diff
@@ -0,0 +1,614 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "boxplot = x.boxplot(column=['size'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Средняя длинна твита"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "78.92663887704397"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "averageNum = x.iloc[:, 1].mean()\n",
+    "averageNum"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Медианная длинна твита"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "73.0"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "medianNum = x.iloc[:, 1].median()\n",
+    "medianNum"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Среднеквадратичное отклонение"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "29.613165284084694"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "MeanSquare = np.std(x.iloc[:, 1])\n",
+    "MeanSquare"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Подсчёт n-грамм слов. 10 самый популярных. Вот тут странно: видимо люди пишут смайлики через пробел, типа \") ) )\" поэтому каждая скобочка считается отдельным словом. Похоже надо нормализовывать текст, а смайлики и тд обрабатывать отдельно(так делали в статье которую я чиатал)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[(\"(')', ')', ')')\", 8708),\n",
```

Comment:
Let's catch user emotions :) At the first time, we an use regular expressions for it :)